lib/login_page.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'book_provider.dart';
+import 'main.dart';
+
+///Login Page Runner
+class LoginPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) => MaterialApp(
+      title: 'Book Store Login',
+      home: Scaffold(
+        body: LoginPageState(),
+      ));
+}
+
+///Login Page State
+class LoginPageState extends StatefulWidget {
+  @override
+  LoginPageComponents createState() => LoginPageComponents();
+}
+
+///Login Page UI
+class LoginPageComponents extends State<LoginPageState> {
+  ///text style
+  TextStyle textStyle = TextStyle(fontSize: 20, color: Colors.white);
+
+  @override
+  Widget build(BuildContext context) {
+    //row for logo
+    final logoDisplay = Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          Icon(
+            Icons.library_books,
+            size: 90,
+          )
+        ]);
+
+    //row for title
+    final titleDisplay = Row(children: <Widget>[
+      Expanded(
+        child: Text(
+          'BTR490 Books',
+          textAlign: TextAlign.center,
+          style: TextStyle(fontSize: 30, fontWeight: FontWeight.w700),
+        ),
+      )
+    ]);
+
+    //textfield for email
+    final emailInput = TextField(
+      obscureText: false,
+      decoration: InputDecoration(
+        contentPadding: const EdgeInsets.fromLTRB(20, 15, 20, 15),
+        hintText: 'Username / Email',
+      ),
+    );
+
+    //textfield for password
+    final passwordInput = TextField(
+      obscureText: true,
+      decoration: InputDecoration(
+        contentPadding: const EdgeInsets.fromLTRB(20, 15, 20, 15),
+        hintText: 'Password',
+      ),
+    );
+
+    //login button
+    final loginButton = Material(
+      elevation: 5,
+      color: Colors.blue,
+      child: MaterialButton(
+          minWidth: MediaQuery.of(context).size.width,
+          padding: const EdgeInsets.fromLTRB(20, 15, 20, 15),
+          onPressed: () {
+            Navigator.push<dynamic>(
+                context,
+                MaterialPageRoute<dynamic>(
+                  builder: (context) => BookProvider(child: ShoppingCartApp()),
+                ));
+          },
+          child: Text(
+            'Login',
+            textAlign: TextAlign.center,
+            style: textStyle,
+          )),
+    );
+
+    return Scaffold(
+      body: Center(
+        child: Container(
+          color: Colors.white,
+          child: Padding(
+            padding: const EdgeInsets.all(36),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                logoDisplay,
+                const SizedBox(height: 15),

```

Comment:
Instead of `SizedBox` can you use `Padding` instead. They are the same as far as performance goes but since we have padding in other places we should stay consistent.